docs/sphinx/dev_guide/release_process.rst

```diff
@@ -24,19 +31,19 @@ The RAJA release process typically involves the following sequence of steps:
      into the **main branch.** When it is approved and all CI checks pass,
```

Comment:
Just to clarify, we create a release candidate branch from `develop`, but merge it into `main`?

Reply:
Thanks, so `main` only has updates up until the latest release.

Reply:
that is correct.